packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -171,6 +171,22 @@ - (void)onMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     }
   } else if ([call.method isEqualToString:@"map#waitForMap"]) {
     result(nil);
+  } else if ([call.method isEqualToString:@"map#takeSnapshot"]) {
+    if (_mapView != nil) {
+      UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
+      format.scale = [[UIScreen mainScreen] scale];
+      UIGraphicsImageRenderer* renderer =
+          [[UIGraphicsImageRenderer alloc] initWithSize:_mapView.frame.size format:format];
+
+      UIImage* image = [renderer imageWithActions:^(UIGraphicsImageRendererContext* context) {
+        [_mapView.layer renderInContext:context.CGContext];
+      }];
+      result([FlutterStandardTypedData typedDataWithBytes:UIImagePNGRepresentation(image)]);
+    } else {
+      result([FlutterError errorWithCode:@"GoogleMap uninitialized"
```

Comment:
ditto: escape early

Reply:
ditto: it's the same style as other code.